internal/service/lexv2models/bot.go

```diff
@@ -256,6 +256,12 @@ func (r *resourceBot) Read(ctx context.Context, req resource.ReadRequest, resp *
 	state.Description = flex.StringToFramework(ctx, out.Description)
 	state.IdleSessionTTLInSeconds = flex.Int32ToFramework(ctx, out.IdleSessionTTLInSeconds)
 
+	members, _ := flattenMembers(ctx, out.BotMembers)
+	if resp.Diagnostics.HasError() {
+		return
+	}
```

Comment:
We should handle the error diagnostics that are returned.

```suggestion
	members, errDiags := flattenMembers(ctx, out.BotMembers)
	resp.Diagnostics.Append(errDiags...)
	if resp.Diagnostics.HasError() {
		return
	}
```